include/FabMember.h

```diff
@@ -0,0 +1,40 @@
+#ifndef MEMBER_CARD_H
+#define MEMBER_CARD_H
+
+#include <cstdint>
+#include <string>
+
+namespace card
+{
+  typedef uint32_t uid_t;
+}
+
+class FabMember
+{
+private:
+  card::uid_t member_uid = 0;
+  std::string holder_name;
+
+public:
+  FabMember();
+  ~FabMember() = default;
+
+  void setUidFromArray(const uint8_t *uid); // tested
```

Comment:
renamed FabMember to FabUser as users may not be actually members
made it a struct to avoid memory management troubles and copy constructors
added a bool to indicate whether it is a valid authenticated user